### DISCUSSION TITLE
Projects (beta): Update issues when the status is changed

### DISCUSSION THREAD

**Discussion by jesuslinares** [community/community#15257] [Projects and Issues]
The new projects are great, but we are facing a little issue.

In the old projects, when we move the issue from one column (_ToDo_) to another (_In progress_), the issue is updated with:`<user> moved this from To Do to In progress in <project name> on <date>`.

For the new projects, this is not happening so when we review an issue we don't have all the information. For example, we don't know who moved the issue from "Pending Review" to "Approved". 

It would be great if the status is added to the issue automatically like it is happening for the old projects.

**Comment by blynkkk** [comment]
Also, it would be great to indicate status change within the issue timeline

For example, when label is added, it's recorded as an event on the issue timeline

<img width="796" alt="image" src="https://user-images.githubusercontent.com/11541426/168301133-ac51bc3d-57c7-41b2-a644-b57d15c12f70.png">

It would be helpful to show a status change in the same manner. This will allow seeing when someone takes it into work, marks as done, etc.

**Comment by snaow** [comment]
Huge +1!

**Comment by tpak** [comment]
This is a duplicate of https://github.com/orgs/community/discussions/5859#discussioncomment-3674139

The latest info was actually updated today: https://github.com/orgs/community/discussions/5859#discussioncomment-3674135
